ciphers/playfair_cipher.c

```diff
@@ -0,0 +1,198 @@
+/**
+ * @file
+ * @brief: Playfair cipher was the first practical digraph substitution cipher
+ * @details: The scheme was invented in 1854 by Charles Wheatstone but was named after Lord Playfair who promoted the use of the cipher.
+ * In playfair cipher unlike traditional cipher we encrypt a pair of alphabets(digraphs) instead of a single alphabet
+ * \author [Rishabh Patel](https://github.com/Rishabhpatel803)
+ */
+
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+#define MX 5
```

Comment:
If using macros, please use descriptive names of at least 5 characters long. Something like `KEY_SIZE`?